client/src/assets/scss/foundation/_settings.scss

```diff
@@ -877,6 +877,7 @@ $topbar-submenu-background: $topbar-background;
 $topbar-title-spacing: 0.5rem 1rem 0.5rem 0;
 $topbar-input-width: 200px;
 $topbar-unstack-breakpoint: medium;
+$topbar-center-logo-bg: #2c3840;
```

Comment:
I do not recommend editing this Foundation settings file - these styles can be overwritten in your custom CSS to maintain on source of truth. This settings file should be the one source of truth for Foundation styles, and anything else custom belongs in something like your `main.scss` file